<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationChooser</class>
 <widget class="QWidget" name="ConfigurationChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>305</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Level</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboLevel"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="chkSetExplicitly">
        <property name="text">
         <string>Set Explicitly</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Milliseconds Width</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Rollout Size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="txtFormat"/>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="buttonBuildFormat">
        <property name="toolTip">
         <string>Build format...</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QCheckBox" name="chkToStandardOutput">
        <property name="text">
         <string>To Standard Output</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QCheckBox" name="chkToFile">
        <property name="text">
         <string>To File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QCheckBox" name="chkEnabled">
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="4">
       <widget class="QLineEdit" name="txtFilename"/>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="QSpinBox" name="spnMillisecondsWidth">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QComboBox" name="cboRollOutSizeUnit"/>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QDoubleSpinBox" name="spnRollOutSize">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>264375895.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="5">
       <widget class="QCheckBox" name="chkPerformanceTracking">
        <property name="text">
         <string>Performance Tracking Features</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
